jekyll/_cci2/configuration-reference.md

```diff
@@ -516,6 +527,16 @@ jobs:
       ... // other config
 ```
 
+You may also use the `machine` class to configure a [runner instance](https://circleci.com/docs/2.0/runner-overview/#section=configuration).
+
+For example:
+
+```yaml
+jobs:
+  job_name:
+    machine: true
+    resource_class: my-namespace/my-runner
+
```

Comment:
You're missing the closing back ticks for this code block.

Reply:
fixed